bodhi/server/models.py

```diff
@@ -2373,7 +2373,8 @@ def get_bug_karma(self, bug):
             positive karma.
         """
         good, bad, seen = 0, 0, set()
-        for comment in reversed(self.comments):
+        # Reversing a generator object {self.comments_since_karma_reset}
+        for comment in reversed([_ for _ in self.comments_since_karma_reset]):
```

Comment:
If you check `self.comments_since_karma_reset` is already reversing the order of the comments. So it should not be needed here.

https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/models.py#L1890